pom.xml

```diff
@@ -1081,5 +1081,10 @@
 			<version>4.0.42.Final</version>
 			<scope>provided</scope>
 		</dependency>
+        <dependency>
+            <groupId>net.sf.py4j</groupId>
+            <artifactId>py4j</artifactId>
+            <version>0.10.9</version>
+        </dependency>
```

Comment:
I do not know how we want to handle this during merge (0.2.0 prep release should anyway include py4j)

Reply:
It is indented using spaces, instead of tabs

Reply:
Oh yes, will quickly fix.